spec/hyrax/transactions/apply_change_set_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+require 'spec_helper'
+require 'hyrax/transactions'
+require 'hyrax/specs/spy_listener'
+require 'dry/container/stub'
+
+RSpec.describe Hyrax::Transactions::ApplyChangeSet do
+  subject(:tx)     { described_class.new }
+  let(:change_set) { Hyrax::ChangeSet.for(resource) }
+  let(:resource)   { build(:hyrax_work) }
+  let(:user)       { create(:user) }
+  let(:xmas)       { DateTime.parse('2018-12-25 11:30').iso8601 }
+
+  before { allow(Hyrax::TimeService).to receive(:time_in_utc).and_return(xmas) }
+
+  describe '#call' do
+    it 'is a success' do
```

Comment:
I believe lines 17 through 25 would benefit with a context scope similar to line 55 (eg. `context "with a valid changeset" do`)